libraries/Microsoft.Bot.Builder/FrameDefinition.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.Bot.Builder
+{
+    public class FrameDefinition
+    {
+        public string Scope { get; set; }
+
+        public string NameSpace { get; set; }
+
+        public List<SlotDefinition> SlotDefinitions { get; set; } = new List<SlotDefinition>();
```

Comment:
I would not personally allow a `List<T>` property to be `set`-able instead preferring to expose as read-only and prefer callers to just use the `List<T>` API to mutate the contents.